src/packages/documents/documents/entity-actions/culture-and-hostnames/manifests.ts

```diff
@@ -16,6 +16,12 @@ const entityActions: Array<ManifestEntityAction> = [
 			repositoryAlias: UMB_DOCUMENT_DETAIL_REPOSITORY_ALIAS,
 			entityTypes: [UMB_DOCUMENT_ENTITY_TYPE],
 		},
+		conditions: [
+			{
+				alias: 'Umb.Condition.UserPermission',
+				verbs: ['Umb.Document.CultureAndHostnames'],
```

Comment:
typically for the other conditions we use 'match' or 'anyOf' for the matching kind of conditions.

Reply:
or allOf